src/locales/en.json

```diff
@@ -17,6 +17,7 @@
   "Categories": {
     "cozy": "Cozy apps",
     "partners": "Partners apps",
+    "ptnb": "expPTNB",
```

Comment:
Yes I am totally sure, this category is aimed to be used only in French. Don't forget to avoid over-quality ;)